workspace/file_resource.go

```diff
@@ -146,6 +146,15 @@ func FileContentSchema(extra map[string]*schema.Schema) map[string]*schema.Schem
 						},
 					}
 				}
+				if strings.HasPrefix(v, "Volume") {
+					return diag.Diagnostics{
+						{
+							Summary:       "Path should start with /Volumes",
+							Severity:      diag.Error,
+							AttributePath: p,
+						},
+					}
+				}
```

Comment:
Hmmm, do we really have that invalid usage?

Reply:
Yes, if path start with Volume instead of /Volumes the it throws the error.